### DISCUSSION TITLE
Add ENV for app description

### DISCUSSION THREAD

**Discussion by DenisIlyushin** [outline/outline#4685] [Ideas]
Would be useful for customization purposes to change general description via .ENV. (If i was right to recognize [place](https://github.com/outline/outline/blob/445d19f43ea8bf822a25518b4e3797cd4cca4052/server/routes/app.ts#L68) in code base)

It would effect the way link is viewed in extended dialog box in telegram when user shares it with anothers.

I see that APP name could be customized, than why not description?
Would be greatful for that small addition!

**Comment by tommoor** [comment]
The app name cannot be customized either in the OSS edition I'm afraid, the Outline branding is required. If you share specific documents with other users the document title and content will be used for social previews though

**Reply by DenisIlyushin** [reply]
Thank you for reply! Understood. You can close this ticket if needed.
